subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -2398,15 +2398,23 @@ static inline int rx_demux_rx(memq_link_t *link, struct node_rx_hdr *rx)
 		}
 
 		ull_scan_aux_setup(link, rx);
-	}
-	break;
+	} break;
```

Comment:
I guess this is your automated tool's work?

I am not sure if this follows the coding guideline of
> Note that the closing brace is empty on a line of its own, except in the cases where it is followed by a continuation of the same statement, ie a while in a do-statement or an else in an if-statement:

(from https://www.kernel.org/doc/html/v4.10/process/coding-style.html)

In either way, this line should probably not have been changing in this PR.

Reply:
Yes it was clang-format change. I've moved it to new line.